zhusuan/distributions/multivariate.py

```diff
@@ -29,6 +33,158 @@
 ]
 
 
+class MultivariateNormalTriL(Distribution):
```

Comment:
Maybe `MultivariateNormalCholesky` is better?

Reply:
`tf.contrib.distributions` replaced `MVNCholesky` to `MVNTriL` previously. Should we align with them?

Reply:
Nope. `tf.contrib` changes frequently and often major changes will happen before things move from the contrib to the core. And from what I see it's still hard to hope `tf.distributions` up-to-date (now it doesn't include multivariate normal, gumbel-softmax and many others). We'd better maintain our own distributions for some time (maybe long) until `tf.distributions` eventually gets complete and stable.

Reply:
I see. I'll change this.